Simplenote/SimplenoteAppDelegate.m

```diff
@@ -421,13 +423,13 @@ - (void)handleWindowDidBecomeMainNote:(NSNotification *)notification
 - (BOOL)splitView:(NSSplitView *)splitView shouldHideDividerAtIndex:(NSInteger)dividerIndex
 {
     // Tag List: Don't draw separators
-    return (dividerIndex == SPSplitViewSectionTags);
+    return (dividerIndex == SPSplitViewSectionTags) || (dividerIndex == SPSplitViewSectionList);
 }
 
 - (NSRect)splitView:(NSSplitView *)splitView effectiveRect:(NSRect)proposedEffectiveRect forDrawnRect:(NSRect)drawnRect ofDividerAtIndex:(NSInteger)dividerIndex
 {
     // Tag List: Don't draw separators
-    return (dividerIndex == SPSplitViewSectionTags) ? CGRectZero : proposedEffectiveRect;
+    return (dividerIndex == SPSplitViewSectionTags) || (dividerIndex == SPSplitViewSectionList && self.noteListViewController.isCollapsed) ? CGRectZero : proposedEffectiveRect;
```

Comment:
`noteListViewController.isCollapsed` doesn't exist. Did you forget to commit your changes to `NoteListViewController`?